diff --git a/src/runtime/src/libraries/Native/Unix/System.IO.Compression.Native/extra_libs.cmake b/src/runtime/src/libraries/Native/Unix/System.IO.Compression.Native/extra_libs.cmake
index f4c7dcd39..6eee6181b 100644
--- a/src/runtime/src/libraries/Native/Unix/System.IO.Compression.Native/extra_libs.cmake
+++ b/src/runtime/src/libraries/Native/Unix/System.IO.Compression.Native/extra_libs.cmake
@@ -9,6 +9,7 @@ macro(append_extra_compression_libs NativeLibsExtra)
       set(ZLIB_LIBRARIES z m)
   else ()
       find_package(ZLIB REQUIRED)
+      set(ZLIB_LIBRARIES ${ZLIB_LIBRARIES} m)
   endif ()
   list(APPEND ${NativeLibsExtra} ${ZLIB_LIBRARIES})
 endmacro()
